ssz-rs/src/merkleization/multiproofs/mod.rs

```diff
@@ -0,0 +1,127 @@
+mod generalized_index;
+
+pub use generalized_index::*;
+
+use crate::merkleization::Node;
+use sha2::{Digest, Sha256};
+use std::collections::{HashMap, HashSet};
```

Comment:
Use BTreeSet and BTreeMap in `alloc` crate for no_std

Reply:
thanks! I may not merge this in right now but ill make an issue at the least